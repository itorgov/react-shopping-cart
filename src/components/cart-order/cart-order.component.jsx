import React from 'react';

import {OrderItems} from "../order-items/order-items.component";
import {OrderInfo} from "../order-info/order-info.component";

import styles from './cart-order.module.scss';

export const CartOrder = () => (
    <div className={styles.order}>
        <OrderItems/>
        <OrderInfo/>
    </div>
);
