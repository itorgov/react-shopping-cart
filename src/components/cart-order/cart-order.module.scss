.order {
  flex: 2;

  padding: 6rem 3rem 6rem 10rem;

  @media only screen and (max-width: 56.25em) {
    padding: 3rem;
  }

  @media only screen and (max-width: 43.75em) {
    flex: 0;
  }
}
