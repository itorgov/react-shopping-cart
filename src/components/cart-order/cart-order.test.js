import React from "react";
import {shallow} from "enzyme";

import {CartOrder} from './cart-order.component';

describe('<CartOrder />', () => {
    it('expect to render', () => {
        expect(shallow(<CartOrder />)).toMatchSnapshot();
    });
});
