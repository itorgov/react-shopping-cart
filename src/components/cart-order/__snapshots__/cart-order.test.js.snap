// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CartOrder /> expect to render 1`] = `
<div
  className="order"
>
  <OrderItems />
  <AddItemModal />
  <OrderInfo />
</div>
`;
